README.md

```diff
@@ -1 +1,235 @@
-# IoT-STH
+#<a id="section0"></a> IoT-STH
+
+* [Introduction] (#section1)
+* [Dependencies](#section2)
+* [Installation](#section3)
+* [Inserting data (single events and aggregated data) into the database](#section4)
+* [Querying data from the database](#section5)
+* [Running the STH server](#section6)
+* [Contact](#section7)
+
+##<a id="section1"></a> Introduction
+The STH component is a FIWARE component in charge of providing aggregated time series information about the evolution of
+entity attribute values in time.
+
+The aggregated time series information is stored in a MongoDB instance. This information can be generated by 2 main
+means:
+
+1. The STH component can directly subscribe to the Context Broker to receive notifications when the entity attribute
+values change, calculating the aggregated time series information and storing it in the MongoDB instance.
+This option is called the minimalist option.
+2. A new sink can be enabled in the Cygnus component to calculate and to update the aggregated time series information
+as the entity attribute values change over time. This option is the formal one.
+
+Regarding the aggregated time series information provided by the STH component, there are 4 main concepts which are
+important to know about:
+
+* <b>Range</b>: The period of time about which the aggregated time series information is provided. Possible valid
+ranges values are: year, month, day, hour, minute.
+* <b>Resolution</b> or <b>aggregation period</b>: The time period by which the aggregated time series information is grouped.
+Possible valid resolution values are: month, day, hour, minute and second. For the time being, we only consider the
+following range-resolution pairs: year-month, month-day, day-hour, hour-minute and minute-second.
+* <b>Origin</b>: For certain range-resolution pair, it is the origin of time for which the aggregated time series
+information applies. For example, for a pair hour-minute, a valid origin value could be: ``2015-03-01T13:00:00.000Z```,
+meaning the 13th hour of March, the 3rd, 2015. The origin is stored using UTC time to avoid locale issues.
+* <b>Offset</b>: For certain range-resolution pair, it is the offset from the origin for which the aggregated time series
+information applies. For example, for a pair hour-minute and an origin ```2015-03-01T13:00:00.000Z```, an offset of 10
+refers to the 10th minute of the concrete hour pointed by the origin. In this example, there would be a maximum of 60
+offsets from 0 to 59 corresponding to each one of the 60 minutes within the concrete hour.
+* <b>Samples</b>: For a quadruple range-resolution-origin-offset, it is the number of samples, values, events or notifications available.
+
+The STH component exposes an HTTP ReST API to let external clients query this aggregated time series information. A
```

Comment:
Fixed in https://github.com/telefonicaid/IoT-STH/commit/553c8aa81a09b7fbd880885e8db989946e1041e6